blog/2022-05-26-caching-api-requests-with-long-lived-workflows.md

```diff
@@ -89,8 +91,7 @@ export async function exchangeRatesWorkflow(): Promise<any> {
     const tomorrow = new Date(today);
     tomorrow.setHours(12, 0, 0, 0);
     tomorrow.setDate(tomorrow.getDate() + 1);
-    // @ts-ignore
-    await sleep(tomorrow - today);
+    await sleep(tomorrow.valueOf() - today.valueOf());
```

Comment:
Also could abstract this to some helper function that doesn't need to be implemented for this post